tests/test_networks.py

```diff
@@ -19,6 +19,11 @@
         'postgres://user:pass@localhost:5432/app',
         'postgres://just-user@localhost:5432/app',
         'postgresql+psycopg2://postgres:postgres@localhost:5432/hatch',
+        'postgresql+asyncpg://user:pass@localhost:5432/app',
+        'postgresql+pg8000://user:pass@localhost:5432/app',
+        'postgresql+psycopg2cffi://user:pass@localhost:5432/app',
+        'postgresql+py-postgresql://user:pass@localhost:5432/app',
+        'postgresql+pygresql://user:pass@localhost:5432/app',
```

Comment:
All those tests would already be valid without your PR. You probably want to parametrize `test_postgres_dsns`

Reply:
I did that but I'm sure if my implementation was good. It's wonderful anyone can review it. :)